main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_data in players_data:
+        user = players_data[player_data]
+        skills = players_data[player_data]["race"]["skills"]
+
+        Race.objects.get_or_create(
+            name=user["race"]["name"],
+            description=user["race"]["description"]
+        )
+        race_id = Race.objects.get(name=user["race"]["name"])
+
+        if user["guild"]:
+            Guild.objects.get_or_create(
+                name=user["guild"]["name"],
+                description=user["guild"]["description"]
+            )
+            guild = Guild.objects.get(name=user["guild"]["name"])
```

Comment:
Similar to the `race_id` assignment, the `guild` variable is assigned the result of `Guild.objects.get(name=user['guild']['name'])`. Consider using the object returned by `get_or_create` to avoid an additional database query.